packages/flutter_tools/lib/src/commands/install.dart

```diff
@@ -15,9 +16,18 @@ class InstallCommand extends FlutterCommand with DeviceBasedDevelopmentArtifacts
     requiresPubspecYaml();
     usesDeviceUserOption();
     usesDeviceTimeoutOption();
-    argParser.addFlag('uninstall-only',
-      help: 'Uninstall the app if already on the device. Skip install.',
-    );
+    argParser
+      ..addFlag(
+        'uninstall-only',
+        help: 'Uninstall the app if already on the device. Skip install.',
+      )
+      ..addOption(
```

Comment:
can you make this a helper function in flutter_command.dart? https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/runner/flutter_command.dart#L203